### DISCUSSION TITLE
No possible action on the display area

### DISCUSSION THREAD

**Discussion by mcamOrng** [gephi/gephi#2839] [Q&A]
Hello,
I am looking for a simple tool to graphically represent the complexity of adhesions between all the activities of my department. I think gephi is adapted to that.
I installed version 0.10 but I have a problem.
When I created my project, I imported the nodes and edges files and the preview showed me the nodes and arrows.
I saved the project.
The next time I open, I no longer see nodes or links. The display area is grayed out and I have no possible action.
Can you help me, please?
Best regards
Martine

**Comment by benafrant** [comment]
xcxvvb

**Comment by benafrant** [comment]
xcxvvb 
> > l hbijkmm.v
# mm n挼，k
